README.md

```diff
@@ -1,2 +1,22 @@
 # Build
-Building files for Galasa
+This repository contains resources that are used to build Galasa, and includes tools such as Docker, Eclipse and Jenkins.
```

Comment:
The vast majority of the build has been completed before we get to this repo.

The main purpose of this is to combine all the built projects into docker images and eclipse update site.

Reply:
updates complete